plugins/dev-tools/package.json

```diff
@@ -38,7 +38,9 @@
   ],
   "dependencies": {
     "@types/dat.gui": "^0.7.5",
-    "dat.gui": "^0.7.7"
+    "chai": "^4.2.0",
+    "dat.gui": "^0.7.7",
+    "mocha": "^7.1.0"
```

Comment:
Mocha is already a dependency of `dev-script`, unclear why it needs to be a dependency here?

Reply:
It seems to build fine without this mocha dependency. I thought this would be needed because the test helpers were compiled and built, as opposed to just being run by dev-script.